src/utils/dates/index.ts

```diff
@@ -0,0 +1,86 @@
+export enum TimeFrame {
+  all = 'all',
```

Comment:
Isn't it easier to call this `present` or `everything`? Not sure what all means 🤔

Reply:
True, I have no beter naming suggestion so this is fine 😄